deployment/base/hg-deployment.yaml

```diff
@@ -131,3 +131,33 @@ spec:
           items:
           - key: hgweb.hgrc
             path: hgweb.hgrc
+
+      initContainers:
+      - name: init-repo-structure
+        securityContext:
+          runAsUser: 33
+          runAsGroup: 33 # www-data
+        image: busybox:1.36.1
+        command:
+          - 'sh'
+          - '-c'
+          - |
+            cd /repos
+            for folder in /repos/[a-z0-9]?*/; do
```

Comment:
This glob means first letter alphanum, second letter non-empty but can be anything, third and subsequent letters can be empty. So it will pick up all folders with names of two or more characters, that start with a-z or 0-9. I checked production and that glob matches all the repo folders there. The reason I changed this glob is so that we don't have to compare `$project` to `$first_letter` any more: `$project` is now guaranteed to be two characters or more.